hedera-node/src/main/java/com/hedera/services/legacy/crypto/SignatureStatus.java

```diff
@@ -163,7 +163,6 @@ public String toLogMessage() {
       case SCHEDULED_TRANSACTION_NOT_IN_WHITELIST:
       case GENERAL_PAYER_ERROR:
       case GENERAL_TRANSACTION_ERROR:
-      case KEY_COUNT_MISMATCH:
```

Comment:
This `ResponseCode` meant a `SignatureList` had a different number of keys from the `KeyList` it was supposed to represent a signature for.

Since `SignatureList`s are no longer usable, it cannot occur.